src/deprecations.jl

```diff
@@ -11,3 +11,64 @@
 # 2.0.0 deprecations
 # Traversals / SP
 @deprecate dists ShortestPaths.distances
+
```

Comment:
This is where we keep most of the deprecations and `const` links to deprecated functions.